backend/src/akvo/lumen/lib/transformation/rename_column.clj

```diff
@@ -12,13 +12,12 @@
   [op-spec]
   (and (util/valid-column-name? (col-name op-spec))
        (string? (new-col-title op-spec))))
-
+;; TODO this affects on how transformations and columns in a dataset_version are related :/
```

Comment:
TODO this affects on how transformations and columns in a dataset_version are related :/